llvm/lib/Target/AMDGPU/VOPCInstructions.td

```diff
@@ -222,6 +222,8 @@ class VOPCInstAlias <VOP3_Pseudo ps, Instruction inst,
 
   let AsmVariantName = AMDGPUAsmVariants.Default;
   let SubtargetPredicate = AssemblerPredicate;
+
+  string DecoderNamespace; // dummy
```

Comment:
Any chance we could do without adding this?

Reply:
I could probably make `VOPCInstAlias` inherit from `LetDummies` instead. Would you prefer that? It is quite a common pattern in our backend.

To remove the dummy completely I would have to abandon the cleanup in e.g. `VOPC_Real_Base` below, where I use one big `let ...` instead five small ones.